### PR TITLE
LPD-26285

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hi,

The issue here is pretty easy to understand: Safari does not handled properly `document.activeElement`. So in case we had more than one submit button in a form we are always sending the first button as `capturedFormButtonElement`.

The solution itself is straightforward: change `document.activeElement` by `event.submitter`.

Testing this is not that easy, I decided to create a Basic Web Content including a form, two submit buttons and some javascript code to print which button was sent as `capturedFormButtonElement`. Then I had to change context to use Safari as the browser and check which button was pressed.

Please, let me know your thoughts and improvements.

Thanks a lot.

### CODE REVIEW

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:sf

File: modules/test/playwright/tests/frontend-js-spa-web/safariSubmitButton.spec.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		await journalEditArticlePage.publishButton.click();
+	});
+
+	await test.step('Set Safari as the browser and check submit buttons in form', async () => {
+		const browser = await webkit.launch();
+
+		const context = await browser.newContext({
+			...safari,
+		});
+
```
Comment (dsanz) [inline modules/test/playwright/tests/frontend-js-spa-web/safariSubmitButton.spec.ts:93]: As we are using a new context here, it's probably wise to dispose it at the end of the step

```
// Dispose context once it's no longer needed.
await context.close();
```

Taken from [here](https://playwright.dev/docs/api/class-browsercontext)

Comment (antonio-ortega) [reply modules/test/playwright/tests/frontend-js-spa-web/safariSubmitButton.spec.ts:93]: Thanks, @dsanz . 

I'll wait to see if there are any more comment and then I'll update it.

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:relevant
ci:test:stable

**Comment by antonio-ortega** [comment]
ci:test:sf
ci:test:relevant
ci:test:stable

**Comment by antonio-ortega** [comment]
ci:test:sf
ci:test:relevant
ci:test:stable

**Comment by antonio-ortega** [comment]
ci:test:relevant
ci:test:stable

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:stable
ci:test:relevant

**Comment by antonio-ortega** [comment]
ci:test:relevant

**Comment by antonio-ortega** [comment]
ci:forward

**Pull request closed without merging**